src/flags.rs

```diff
@@ -31,6 +33,11 @@ impl Flags {
         let date_inputs: Vec<&str> = matches.values_of("date").unwrap().collect();
         let dir_order_inputs: Vec<&str> = matches.values_of("group-dirs").unwrap().collect();
         let blocks_inputs: Vec<&str> = matches.values_of("blocks").unwrap().collect();
+        let ignore_globs_inputs: Vec<&str> = if matches.is_present("ignore-glob") {
+            matches.values_of("ignore-glob").unwrap().collect()
```

Comment:
I think you can keep the same format than the options above here and remove the if/else condition because the `.collect()` will return an empty array if no option is set.

Reply:
For some reason removing the `if` clause results in an error:

```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/libcore/option.rs:378:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

Reply:
The above error happens only when running with out `-I`/`--ignore-glob`.

Reply:
I was able to make it work without an `if` statement by providing a default value in `src/app.rs`.